framework/src/governance/lib/datazone/datazone-gsr-msk-datasource.ts

```diff
@@ -0,0 +1,232 @@
+              actions: [
+                'glue:GetSchemaVersion',
+                'glue:ListSchemas',
+                'glue:ListSchemaVersions',
+              ],
+              resources: [glueRegistryArn, glueRegistrySchemasArn],
+            }),
+            new PolicyStatement({
+              effect: Effect.ALLOW,
+              actions: [
+                'kafka:DescribeClusterV2',
+              ],
+              resources: [clusterArn],
+            }),
+            new PolicyStatement({
+              effect: Effect.ALLOW,
+              actions: [
+                'kafka:ListClustersV2',
+              ],
+              resources: [listClustersArn],
+            }),
+            new PolicyStatement({
+              effect: Effect.ALLOW,
+              actions: [
+                'ssm:GetParameter',
+                'ssm:PutParameter',
+                'ssm:DeleteParameter',
+                'ssm:GetParametersByPath',
+              ],
+              resources: [
+                `arn:${partition}:ssm:${region}:${accountId}:parameter${parameterPrefix}*`,
+              ],
+            }),
+          ],
+        }),
+      },
+    });
+
+    this.dataZoneMembership = new CfnProjectMembership(this, 'ProjectMembership', {
+      designation: 'PROJECT_CONTRIBUTOR',
+      domainIdentifier: props.domainId,
+      projectIdentifier: props.projectId,
+      member: {
+        userIdentifier: this.datasourceLambdaRole.roleArn,
+      },
+    });
+
+    const lambdaCrawler = new Function(this, 'DataZoneGsrMskDataSource', {
+      runtime: Runtime.NODEJS_20_X,
+      handler: 'index.handler',
+      role: this.datasourceLambdaRole,
+      timeout: Duration.minutes(5),
+      code: Code.fromAsset(__dirname + '/resources/datazone-gsr-msk-datasource/'),
+      environment: {
+        DOMAIN_ID: props.domainId,
+        PROJECT_ID: props.projectId,
+        CLUSTER_NAME: props.clusterName,
+        REGION: region,
+        REGISTRY_NAME: props.registryName,
+        ACCOUNT_ID: accountId,
+        PARAMETER_PREFIX: parameterPrefix,
+        PARTITION: partition,
+      },
+    });
+
+    lambdaCrawler.node.addDependency(this.dataZoneMembership);
+
+    // Add EventBridge Rule for cron schedule (if provided)
+    if (props.runSchedule || props.enableSchemaRegistryEvent === undefined) {
+      this.scheduleRule = new Rule(this, 'ScheduledRule', {
+        schedule: props.runSchedule || Schedule.expression('cron(1 0 * * ? *)'),
+        targets: [new LambdaFunction(lambdaCrawler)],
+      });
+    }
+
+    // Add EventBridge Rule for Glue Schema Registry changes (if enabled)
+    if (props.enableSchemaRegistryEvent) {
+      this.createUpdateEventRule = new Rule(this, 'SchemaRegistryEventRule', {
+        eventPattern: {
+          source: ['aws.glue'],
+          detail: {
+            eventSource: ['glue.amazonaws.com'],
+            eventName: ['CreateSchema', 'RegisterSchemaVersion'],
+            responseElements: {
```

Comment:
Can we add a filter to only catch the event for schema registry, i think the current definition also catch the glue tables.